### ISSUE TITLE
Error when running with '-d'

### ISSUE THREAD

**Issue by bararchy** [Ekultek/whitewidow#9]
Using latest version from git:

```
/home/unshadow/Desktop/git-projects/whitewidow/lib/copy.rb:2:in `file': wrong number of arguments (given 0, expected 2) (ArgumentError)
    from whitewidow.rb:483:in `<main>'
```

**Comment by Ekultek** [comment]
Roll back to 1.0.4 for now, I'll fix it when I have time

**Comment by Ekultek** [comment]
Problem is the Copy module takes two parameters, will fix when I have time, simple fix.

**Comment by Ekultek** [comment]
Patched redownload version 1.0.5.1
